ddl/delete_range.go

```diff
@@ -249,12 +267,11 @@ func (ea *elementIDAlloc) alloc() int64 {
 // insertJobIntoDeleteRangeTable parses the job into delete-range arguments,
 // and inserts a new record into gc_delete_range table. The primary key is
 // (job ID, element ID), so we ignore key conflict error.
-func insertJobIntoDeleteRangeTable(ctx context.Context, sctx sessionctx.Context, job *model.Job) error {
+func insertJobIntoDeleteRangeTable(ctx context.Context, sctx sessionctx.Context, job *model.Job, ea *elementIDAlloc) error {
```

Comment:
Why add `elementIDAlloc`?

Reply:
Because the element ID should be unique in the parent job, instead of sub-job.